docs/content/tutorials/kic/ingress-udp-walkthrough.md

```diff
@@ -143,24 +143,24 @@ To route UDP requests to an application in the mesh through the NGINX Plus Ingre
 
 Now that everything for the NGINX Plus Ingress Controller is deployed, we can now send datagrams to the udp-listener application.
 
-3. Use the IP and port defined in the [Install NGINX Plus Ingress Controller](#install-nginx-plus-ingress-controller) section to send a netcat UDP message:
+1. Use the IP and port defined in the [Install NGINX Plus Ingress Controller](#install-nginx-plus-ingress-controller) section to send a netcat UDP message:
```

Comment:
Was restarting the numbered list intentional?

Reply:
Since they're separated by a header, the linter wants it to be a new list.

Reply:
Cool. Just making sure.